pkg/ctl/sender/client_containerd_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 New Relic Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//go:build linux || darwin
+
+package sender
+
+import (
+	"testing"
+
+	"github.com/newrelic/infrastructure-agent/pkg/helpers"
+)
+
+func TestContainerdClient(t *testing.T) {
+	type args struct {
+		containerID string
+		namespace   string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantC   bool
+		wantErr bool
+	}{
+		{"no container id fails", args{containerID: "", namespace: ""}, false, true},
```

Comment:
taking into account that there is only one test and it's for empty containerId, does it make sense to have this test? Will we test it with docker? (or is it meant to be tested locally?)